aspnetcore/fundamentals/configuration/index.md

```diff
@@ -75,11 +75,13 @@ The following applies to apps using the [Generic Host](xref:fundamentals/host/ge
 To remove the providers added by `CreateDefaultBuilder`, call `Sources.Clear()` first:
 
 ```csharp
-.ConfigureAppConfiguration((hostingContext, config) =>
-{
-    config.Sources.Clear();
-    // Add providers here
-})
+Host.CreateDefaultBuilder(args)
```

Comment:
I considered that it's a bit context-less in this section :D Whereas the context was clearer down in the _ConfigureAppConfiguration_ section.